bootstrap.py

```diff
@@ -247,6 +247,18 @@ def get_puppet_version():
             return puppet_version
 
 
+def is_fips():
+    """
+    Checks to see if the system is FIPS enabled.
+    """
+    fipsfile = open("/proc/sys/crypto/fips_enabled", "r")
+    fipsStatus = fipsfile.read(1)
+    if fipsStatus == "1":
+        return True
+    else:
+        return False
```

Comment:
Any reason not to do this?

```python
return fipsStatus == "1"
```

Reply:
It seemed more pythonic to return True/False in lieu of a value. But I can be convinced otherwise.

Reply:
`return fipsStatus == "1"` will return (True/False) just fine and is imho the pythonicst way to do it.

Reply:
And, for those interested, it's the Ruby way also (though I'm struggling to find a link to back myself up).